src/Core/Algorithms/Legacy/Fields/MarchingCubes/MarchingCubes.h

```diff
@@ -29,57 +29,65 @@
 #ifndef CORE_ALGORITHMS_VISUALIZATION_MARCHINGCUBES_H
 #define CORE_ALGORITHMS_VISUALIZATION_MARCHINGCUBES_H 1
 
-#include <Core/Datatypes/Field.h>
-#include <Core/Datatypes/Matrix.h>
+#include <Core/Datatypes/Legacy/Field/Field.h>
+//#include <Core/Datatypes/Matrix.h>
 
-#include <Core/Thread/Thread.h>
-#include <Core/Geom/GeomGroup.h>
+//#include <Core/Thread/Thread.h>
+//#include <Core/Geom/GeomGroup.h>
 
 #include <string>
 #include <vector>
 
-#include <Core/Algorithms/Util/AlgoBase.h>
-#include <Core/Algorithms/Fields/share.h>
-
-namespace SCIRunAlgo {
+#ifdef SCIRUN4_CODE_TO_BE_ENABLED_LATER
+ #include <Core/Algorithms/Util/AlgoBase.h>
+#endif
 
-using namespace SCIRun;
+#include <Core/Algorithms/Base/AlgorithmBase.h>
+#include <Core/Algorithms/Field/share.h>
```

Comment:
Wrong share.h for this file (Windows build fails due to this): use Core/Algorithms/Legacy/Fields/share.h instead